apis/clusters/v1beta1/kafkaconnect_types.go

```diff
@@ -504,7 +522,7 @@ func (tc *TargetCluster) AreManagedClustersEqual(mClusters []*ManagedCluster) bo
 
 	for i, mCluster := range mClusters {
 		cluster := tc.ManagedCluster[i]
-		if *mCluster != *cluster {
+		if mCluster.KafkaConnectVPCType != cluster.KafkaConnectVPCType {
```

Comment:
Why don't you check other fields?

Reply:
because other fields and in case when clusterref is filled it may cause webhook fail

Reply:
This issue is still not fixed?